src/protocols/rdp/settings.c

```diff
@@ -860,7 +860,7 @@ guac_rdp_settings* guac_rdp_parse_args(guac_user* user,
     /* Timezone if provided by client, or use handshake version */
```

Comment:
Is this comment about the handshake version wrong now?

Reply:
Yeeeessss ... I'll correct that as part of the relevant commit.